### PR TITLE
Replace clusterworkspaces/{content,workspace} subresources in RBAC with workspaces{/content,}

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: pkg/virtual/workspaces/registry/rest_test.go [hunk 1]
```diff
@@ -361,64 +358,6 @@ func TestListWorkspacesWithUserPermission(t *testing.T) {
 	applyTest(t, test)
 }
 
-func TestListWorkspacesForbidden(t *testing.T) {
```
Comment (davidfestal) [inline pkg/virtual/workspaces/registry/rest_test.go:364]: Test are removed here, but I didn't see any test added to the new authorizer added in the `build.go` file. Did I miss it ?
Do you think there should be one ?

**Review by davidfestal — COMMENTED** [review]
Do we want to mention (in the title or description) the move to a real authorizer.
That seems quite an important change in this PR

**Comment by sttts** [comment]
> Do we want to mention (in the title or description) the move to a real authorizer.

You missed https://github.com/kcp-dev/kcp/pull/1710.

File: pkg/virtual/workspaces/registry/rest_test.go (hunk 1 above)
Comment (sttts) [reply pkg/virtual/workspaces/registry/rest_test.go:364]: added get/delete to e2e tests. List everybody with access has now.

Comment (davidfestal) [reply pkg/virtual/workspaces/registry/rest_test.go:364]: OK. Thanks !

**Review by davidfestal — APPROVED** [review]

**Pull request merged**